meta-xt-control-domain/recipes-guest/doma/files/doma-unpause.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Android VM unpause service
+
+Requires=backend-ready@domd-qemu-monitor.service doma-restart-monitor.service
+After=backend-ready@domd-qemu-monitor.service
+
+[Service]
+Type=simple
+
+ExecStart=/bin/bash -c 'echo "[doma-unpause]: Unpausing DomA" && /usr/sbin/xl -v unpause DomA && echo "[doma-unpause]: Unpaused DomA"'
+
+Restart=always
+RemainAfterExit=true
```

Comment:
> RemainAfterExit=
>Takes a boolean value that specifies whether the service shall be considered active even when all its processes exited.

Is this a correct option for this service?

Reply:
Hi @lorc,

Done for doma-unpause.service and domu-unpause.service